src/mapml/layers/FeatureLayer.js

```diff
@@ -130,7 +130,7 @@ export var FeatureLayer = L.FeatureGroup.extend({
         M._metaContentToObject(
           mapml.querySelector('map-meta[name=cs]').getAttribute('content')
         ).content) ||
-      'PCRS';
```

Comment:
This causes problems with the GeoServer MapML module, because the _getNativeVariables: function (mapml) argument is actually just the feature, not the whole document, AFAICT.  So, even though there's a `<map-meta name="cs" content="pcrs"></map-meta>` in that document, it's not actually in the `<map-feature>` element, it's somewhere above it, earlier in the document.  Given that, we either have to change the GeoServer module to emit the cs attribute on the `<map-geometry>` instead of as a <map-meta> element, or change what gets passed to this function to include the whole MapML response, or add another parameter for cs.

Reply:
As per our discussion, I have added the map-meta's that may come from the mapml document to the shadowRoot of map-extent along with the map-feature, they are used accordingly also by the zoom to here link.